gtsam/hybrid/GaussianMixture.h

```diff
@@ -256,6 +256,10 @@ class GTSAM_EXPORT GaussianMixture
   /// Check whether `given` has values for all frontal keys.
   bool allFrontalsGiven(const VectorValues &given) const;
 
+  /// Helper method to compute the error of a conditional.
+  double conditionalError(const GaussianConditional::shared_ptr &conditional,
```

Comment:
Static? Fix in next PR…

Reply:
It needs the logConstant_ member, so can't be static. I did mark it private though.